cmsdb/processes/ewk.py

```diff
@@ -6,36 +6,40 @@
 
 __all__ = [
     "dy",
-    "dy_lep",
-    "dy_lep_m4to10",
-    "dy_lep_m10to50",
-    "dy_lep_m50", "dy_lep_m50_0j", "dy_lep_m50_1j", "dy_lep_m50_2j", "dy_lep_m50_3j", "dy_lep_m50_4j",
-    "dy_lep_0j", "dy_lep_1j", "dy_lep_2j",
-    "dy_lep_m50_ht70to100", "dy_lep_m50_ht100to200", "dy_lep_m50_ht200to400",
-    "dy_lep_m50_ht400to600", "dy_lep_m50_ht600to800", "dy_lep_m50_ht800to1200",
-    "dy_lep_m50_ht1200to2500", "dy_lep_m50_ht2500",
-    "dy_lep_pt0to50", "dy_lep_pt50to100", "dy_lep_pt100to250", "dy_lep_pt250to400",
-    "dy_lep_pt400to650", "dy_lep_pt650",
+    "dy_m4to10",
+    "dy_m10to50",
+    "dy_m50toinf", "dy_m50toinf_0j", "dy_m50toinf_1j", "dy_m50toinf_2j", "dy_m50toinf_ge3j",
+    "dy_m50toinf_3j", "dy_m50toinf_4j",
+    "dy_m50toinf_1j_pt0to40", "dy_m50toinf_1j_pt40to100", "dy_m50toinf_1j_pt100to200", "dy_m50toinf_1j_pt200to400",
+    "dy_m50toinf_1j_pt400to600", "dy_m50toinf_1j_pt600toinf",
+    "dy_m50toinf_2j_pt0to40", "dy_m50toinf_2j_pt40to100", "dy_m50toinf_2j_pt100to200", "dy_m50toinf_2j_pt200to400",
+    "dy_m50toinf_2j_pt400to600", "dy_m50toinf_2j_pt600toinf",
+    "dy_0j", "dy_1j", "dy_2j",
+    "dy_m50toinf_ht70to100", "dy_m50toinf_ht100to200", "dy_m50toinf_ht200to400",
+    "dy_m50toinf_ht400to600", "dy_m50toinf_ht600to800", "dy_m50toinf_ht800to1200",
+    "dy_m50toinf_ht1200to2500", "dy_m50toinf_ht2500toinf",
+    "dy_pt0to50", "dy_pt50to100", "dy_pt100to250", "dy_pt250to400",
+    "dy_pt400to650", "dy_pt650toinf",
     "z",
     "z_nunu",
     "z_nunu_ht100to200", "z_nunu_ht200to400", "z_nunu_ht400to600",
     "z_nunu_ht600to800", "z_nunu_ht800to1200", "z_nunu_ht1200to2500",
-    "z_nunu_ht2500",
+    "z_nunu_ht2500toinf",
     "z_qq",
-    "z_qq_ht200to400", "z_qq_ht400to600", "z_qq_ht600to800", "z_qq_ht800",
+    "z_qq_ht200to400", "z_qq_ht400to600", "z_qq_ht600to800", "z_qq_ht800toinf",
     "w",
     "w_taunu", "w_munu",
     "w_lnu",
     "w_lnu_ht70to100", "w_lnu_ht100to200", "w_lnu_ht200to400", "w_lnu_ht400to600",
-    "w_lnu_ht600to800", "w_lnu_ht800to1200", "w_lnu_ht1200to2500", "w_lnu_ht2500",
+    "w_lnu_ht600to800", "w_lnu_ht800to1200", "w_lnu_ht1200to2500", "w_lnu_ht2500toinf",
```

Comment:
we removed the `lep` tag from dy_lep but not `lnu` from `w_lnu`. Are there any `w_qq` samples?